src/Sulu/Component/Rest/RequestParametersTrait.php

```diff
@@ -25,14 +25,18 @@ trait RequestParametersTrait
      *
      * @param string $name
      * @param bool $force TRUE if value is mandatory
-     * @param mixed $default value if parameter not exists
+     * @param mixed $default value if parameter not exists, this will be ignored if $force is set to true
```

Comment:
would currently keep it as it was, this would more make problems and could crash existing applications.